bson/doc.go

```diff
@@ -4,8 +4,9 @@
 // not use this file except in compliance with the License. You may obtain
 // a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
 
-// Package bson is a library for reading, writing, and manipulating BSON. The
-// library has two families of types for representing BSON.
+// Package bson is a library for reading, writing, and manipulating BSON.
```

Comment:
Consider a one sentence description of BSON, perhaps just copied from https://docs.mongodb.com/manual/reference/bson-types/

Reply:
done.